styles/atom-import-cost.less

```diff
@@ -7,5 +7,5 @@
   opacity: 0.5;
   margin-left: 10px;
   position: relative;
-  top: -(@component-line-height - (@component-padding/2));
+  top: -@component-line-height;
```

Comment:
you have to leave the padding as it is. Since it will disturb the position of the `size` from the top by 2px.